package.json

```diff
@@ -40,7 +40,8 @@
     "grunt-wp-i18n": "^1.0.2",
     "lg-thumbnail": "^1.1.0",
     "lightgallery": "^1.6.8",
-    "time-grunt": "^1.4.0"
+    "time-grunt": "^1.4.0",
+    "responsive-foundation": "git+ssh://git@github.com:bu-ist/responsive-foundation#migrate-off-bower"
```

Comment:
We'll want to update this line to point to a stable version of Foundation before merging this.

Reply:
changed to 3.0.0 after Foundation was updated.